server/mbta_api.py

```diff
@@ -231,11 +237,13 @@ async def initial_request_data(route_ids, test_mode=False):
     )
     sightings = get_recent_sightings_for_lines()
     git_tag = get_git_tag()
+    static_data = get_static_data()
```

Comment:
Could we move this call outside of the `initial_request_data` function? That way we don't have to `open()` this file every time we want to serve the page — we should only need to do it once.